codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2GoAwayFrame.java

```diff
@@ -87,6 +98,11 @@ public Http2GoAwayFrame setExtraStreamIds(int extraStreamIds) {
     }
 
     @Override
+    public int lastStreamId() {
+        return lastStreamId;
+    }
+
+    @Override
     public Http2GoAwayFrame copy() {
         return (Http2GoAwayFrame) super.copy();
```

Comment:
Actually I think this will not result in the correct thingy... You need to ensure the lastStreamId etc is set as well. Which will not be true if you call super.copy() . Event worse I think this will result in a ClassCastException